Modules/ExampleImageInverter/files.cmake

```diff
@@ -0,0 +1,19 @@
+set(CPP_FILES
+  ExampleImageInverter.cpp
+)
+
+set(UI_FILES
+
+)
+
+set(H_FILES
+  include/ExampleImageInverter.h
+)
+
+set(MOC_H_FILES
+  include/ExampleImageInverter.h
```

Comment:
This isn't needed

Reply:
It doesn't need to go through moc, since it isn't qt based.

Reply:
Thanks, I didn't catch that when doing my find+replace. I just removed this in the newest commit.